<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpenConnection</class>
 <widget class="QDialog" name="DialogOpenConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>217</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>217</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>217</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open Connection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="toolTip">
      <string>Will open in a new window instead of the current one.</string>
     </property>
     <property name="text">
      <string>Open connection in a new window</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="autoConnectCheckbox">
     <property name="toolTip">
      <string>Will connect to the selected bookmark when the application is launched.</string>
     </property>
     <property name="text">
      <string>Auto-connect to bookmark on launch</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Bookmark it</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bookmarks:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>DialogOpenConnection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
